tests/test_snapshot_download.py

```diff
@@ -22,36 +24,52 @@ def setUpClass(cls):
         cls._token = cls._api.login(username=USER, password=PASS)
 
     def setUp(self) -> None:
-        repo = Repository(
-            REPO_NAME,
-            clone_from=f"{USER}/{REPO_NAME}",
+        repo_public = Repository(
+            REPO_NAME_PUBLIC,
+            clone_from=f"{USER}/{REPO_NAME_PUBLIC}",
             use_auth_token=self._token,
             git_user="ci",
             git_email="ci@dummy.com",
         )
 
-        with repo.commit("Add file to main branch"):
+        with repo_public.commit("Add file to main branch"):
             with open("dummy_file.txt", "w+") as f:
                 f.write("v1")
 
-        self.first_commit_hash = repo.git_head_hash()
+        self.first_commit_hash = repo_public.git_head_hash()
 
-        with repo.commit("Add file to main branch"):
+        with repo_public.commit("Add file to main branch"):
             with open("dummy_file.txt", "w+") as f:
                 f.write("v2")
             with open("dummy_file_2.txt", "w+") as f:
                 f.write("v3")
 
-        self.second_commit_hash = repo.git_head_hash()
+        self.second_commit_hash = repo_public.git_head_hash()
+
+        repo_private = Repository(
```

Comment:
There's a bit of code duplication here, but I couldn't see a way to convert an existing public repo to private directly with `huggingface_hub`

Reply:
You can use `HfApi().update_repo_visibility` to make the repo private.

https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/hf_api.py#L638

Reply:
Thanks for the tip @osanseviero ! This is much simpler indeed :)